eZ/Publish/Core/Limitation/OwnerLimitationType.php

```diff
@@ -142,7 +142,7 @@ public function evaluate(APILimitationValue $value, APIUserReference $currentUse
          * @var $object ContentInfo
          */
         $isOwner = $object->ownerId === $userId;
-        $isSelf = $object->id === $userId;
+        $isSelf = !$object instanceof ContentCreateStruct && $object->id === $userId;
```

Comment:
would probably be easier to read if it is `$object instanceof ContentInfo && $object->id === $userId`